### ISSUE TITLE
Warn about missing configuration when running "phive install"

### ISSUE THREAD

**Issue by cweiske** [phar-io/phive#430]
When running `phive install` in a directory that does not contain a `.phive/phars.xml` file, it simply exits as if everything is alright.

phive should warn that it does not have any configuration to install something.

This is with phive 0.15.2

```
$ phive install; echo $?
Phive 0.15.2 - Copyright (C) 2015-2024 by Arne Blankerts, Sebastian Heuer and Contributors

0
```

**Comment by theseer** [comment]
Valid Point :)

**Comment by theseer** [comment]
Implemented in 0.15.3
